src/pyFAI/gui/tasks/ExperimentTask.py

```diff
@@ -268,8 +280,11 @@ def __detectorModelUpdated(self):
             self._detectorOrientationValue.setText("")
             self._detectorSensorLabel.setText("")
             self._detectorSensorName.setText("")
+            self._detectorParallax.setChecked(False)
+            self._detectorParallax.setVisible(False)
```

Comment:
IMO it's usually best to enable/disable widgets than to show/hide

Reply:
Noted.

Reply:
Checked in Valentin's code and he is always using `setVisible` in order to remove unused widgets from the display and make it simpler ... I thinks I will stick to his philosophy for calib2...